utils/build-script-impl

```diff
@@ -1240,6 +1240,18 @@ if [[ ! "${SKIP_BUILD_SWIFTPM}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" swiftpm)
 fi
 
+# Checks if a given product is enabled (i.e. part of $PRODUCS array)
+function contains_product() {
```

Comment:
I do have mild preference for %{} but lit doesn't enforce it. I personally use it because I think it makes the behavior more obvious, and I had some idea that one day I might refactor lit to require it so that it could have a more strict shell parsing model. Chances of that actually happening... low. :)